src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should be "2020, 2021" since it is not a new file.

Reply:
Thank you. I have changed that.